airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -234,11 +234,10 @@ public boolean existingSchemaMatchesStreamConfig(final StreamConfig stream,
       tableClusteringMatches = clusteringMatches(stream, standardExistingTable);
       tablePartitioningMatches = partitioningMatches(standardExistingTable);
     }
-    LOGGER.info("Alter Table Report {} {} {} {}; Clustering {}; Partitioning {}",
+    LOGGER.info("Alter Table Report {} {} {}; Clustering {}; Partitioning {}",
             alterTableReport.columnsToAdd(),
             alterTableReport.columnsToRemove(),
             alterTableReport.columnsToChangeType(),
-            alterTableReport.isDestinationV2Format(),
```

Comment:
this is always true b/c we throw an exception above if it's not. removing it from the log message